keylime/dsse/dsse.py

```diff
@@ -50,19 +53,19 @@ def b64enc(m: bytes) -> str:
     return base64.standard_b64encode(m).decode("utf-8")
 
 
-def b64dec(m: str) -> bytes:
-    m = m.encode("utf-8")
+def b64dec(m_bytes: str) -> bytes:
+    m_bytes = m_bytes.encode("utf-8")
 
-    decoded = None
+    decoded: Optional[bytes] = None
     try:
-        decoded = binascii.unhexlify(m)
+        decoded = binascii.unhexlify(m_bytes)
     except binascii.Error:
         pass
     if not decoded:
         try:
-            decoded = base64.b64decode(m, validate=True)
+            decoded = base64.b64decode(m_bytes, validate=True)
         except binascii.Error:
-            decoded = base64.b64decode(m, altchars="-_", validate=True)
+            decoded = base64.b64decode(m_bytes, altchars="-_", validate=True)  # type: ignore
```

Comment:
One of the tools didn't realize that altchars is allowed to be 2 ASCII characters (str), so what else can one do than have it ignore it...